test/stage2/cbe.zig

```diff
@@ -794,6 +794,55 @@ pub fn addCases(ctx: *TestContext) !void {
         });
     }
 
+    {
+        // TODO: move these cases into the programs themselves once stage 2 has array literals
+        // TODO: add u64 tests, ran into issues with the literal generated for std.math.maxInt(u64)
+        var case = ctx.exeFromCompiledC("Wrapping operations", .{});
+        const programs = comptime blk: {
```

Comment:
mind iterating over these at runtime? just to avoid blowing up the compile time of the test harness.

Reply:
will do, I figured that would be the response to this ;)